src/main/java/uk/gov/companieshouse/company/profile/service/CompanyProfileService.java

```diff
@@ -334,44 +259,68 @@ private void setUpdateIfNotNullOtherwiseRemove(Update update, String key, Object
     }
 
     /**
-     * Check if psc statements link exists already on document and
+     * transforms link type string to format required to fetch from db.
+     */
+    private String convertToDBformat(String linkType) {
```

Comment:
Should we look to replace this with another property in a link request? Just in case we ever need to incorporate a DB link that's different to the linkType?

Reply:
dont mind adding this, just concerned it will bloat up the link request class with variables that have the same value